### PR TITLE
feat: add ocaml icons

### PR DESCRIPTION
**Pull request by aripiprazole**

### CODE REVIEW

File: .npmrc
Comment (cadamsdev) [inline .npmrc:1]: Is there a reason this file was added? Did you run into an issue? If not, can you remove it.

**Comment by cadamsdev** [comment]
Hey thanks for creating a PR 😊

Some feedback
- Is there a reason you added the .npmrc file? Did you run into an issue? If not, can you remove that file?

Other than that, changes look good to me. 👍

**Comment by aripiprazole** [comment]
Hello, I just committed the file because it was done on my work computer, sorry

**Comment by cadamsdev** [comment]
> Hello, I just committed the file because it was done on my work computer, sorry

It's all good. Thanks for contributing. 😊

**Pull request merged**